Week2/HW2/5.roundNumber.js

```diff
@@ -0,0 +1,8 @@
+// rounding a number and printing out the highest number
+let z = 7.25;
+console.log(z);
+let a = Math.max(7.25);
+console.log(a);
+
+if (a > z) {let c = a;
```

Comment:
You don't have to do this, `Math.max` is already good enough:) So this code is not needed.